To: vim_dev@googlegroups.com
Subject: Patch 7.3.183
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.183 (after 7.3.174)
Problem:    When Exuberant ctags binary is exuberant-ctags it's not found.
Solution:   Add configure check for exuberant-ctags.
Files:	    src/configure.in, src/auto/configure


*** ../vim-7.3.182/src/configure.in	2011-05-05 18:10:11.000000000 +0200
--- src/configure.in	2011-05-10 15:39:38.000000000 +0200
***************
*** 3375,3384 ****
  dnl -i+m to test for older Exuberant ctags
  AC_MSG_CHECKING(how to create tags)
  test -f tags && mv tags tags.save
! if (eval exctags --version /dev/null | grep Exuberant) < /dev/null 1>&AC_FD_CC 2>&1; then
!   TAGPRG="exctags -I INIT+ --fields=+S"
! elif (eval ctags --version /dev/null | grep Exuberant) < /dev/null 1>&AC_FD_CC 2>&1; then
    TAGPRG="ctags -I INIT+ --fields=+S"
  else
    TAGPRG="ctags"
    (eval etags	   /dev/null) < /dev/null 1>&AC_FD_CC 2>&1 && TAGPRG="etags"
--- 3375,3386 ----
  dnl -i+m to test for older Exuberant ctags
  AC_MSG_CHECKING(how to create tags)
  test -f tags && mv tags tags.save
! if (eval ctags --version /dev/null | grep Exuberant) < /dev/null 1>&AC_FD_CC 2>&1; then
    TAGPRG="ctags -I INIT+ --fields=+S"
+ elif (eval exctags --version /dev/null | grep Exuberant) < /dev/null 1>&AC_FD_CC 2>&1; then
+   TAGPRG="exctags -I INIT+ --fields=+S"
+ elif (eval exuberant-ctags --version /dev/null | grep Exuberant) < /dev/null 1>&AC_FD_CC 2>&1; then
+   TAGPRG="exuberant-ctags -I INIT+ --fields=+S"
  else
    TAGPRG="ctags"
    (eval etags	   /dev/null) < /dev/null 1>&AC_FD_CC 2>&1 && TAGPRG="etags"
*** ../vim-7.3.182/src/auto/configure	2011-05-05 18:10:11.000000000 +0200
--- src/auto/configure	2011-05-10 15:39:41.000000000 +0200
***************
*** 12028,12037 ****
  { $as_echo "$as_me:${as_lineno-$LINENO}: checking how to create tags" >&5
  $as_echo_n "checking how to create tags... " >&6; }
  test -f tags && mv tags tags.save
! if (eval exctags --version /dev/null | grep Exuberant) < /dev/null 1>&5 2>&1; then
!   TAGPRG="exctags -I INIT+ --fields=+S"
! elif (eval ctags --version /dev/null | grep Exuberant) < /dev/null 1>&5 2>&1; then
    TAGPRG="ctags -I INIT+ --fields=+S"
  else
    TAGPRG="ctags"
    (eval etags	   /dev/null) < /dev/null 1>&5 2>&1 && TAGPRG="etags"
--- 12028,12039 ----
  { $as_echo "$as_me:${as_lineno-$LINENO}: checking how to create tags" >&5
  $as_echo_n "checking how to create tags... " >&6; }
  test -f tags && mv tags tags.save
! if (eval ctags --version /dev/null | grep Exuberant) < /dev/null 1>&5 2>&1; then
    TAGPRG="ctags -I INIT+ --fields=+S"
+ elif (eval exctags --version /dev/null | grep Exuberant) < /dev/null 1>&5 2>&1; then
+   TAGPRG="exctags -I INIT+ --fields=+S"
+ elif (eval exuberant-ctags --version /dev/null | grep Exuberant) < /dev/null 1>&5 2>&1; then
+   TAGPRG="exuberant-ctags -I INIT+ --fields=+S"
  else
    TAGPRG="ctags"
    (eval etags	   /dev/null) < /dev/null 1>&5 2>&1 && TAGPRG="etags"
*** ../vim-7.3.182/src/version.c	2011-05-10 14:44:07.000000000 +0200
--- src/version.c	2011-05-10 15:40:48.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     183,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
78. You find yourself dialing IP numbers on the phone.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
